doc/source/api_doc.rst

```diff
@@ -18,7 +18,15 @@ The Galaxy API addresses these and other situations by exposing Galaxy
 internals through an additional interface, known as an Application Programming
 Interface, or API.
 
-Various language specific bindings for interfacing with the Galaxy API have been developed by the Galaxy community including `Python <https://bioblend.readthedocs.io/>`__, `PHP <https://github.com/galaxyproject/blend4php>`__, `Java <https://github.com/jmchilton/blend4j>`__, and `JavaScript <https://github.com/galaxyproject/galaxy/tree/dev/client/src/mvc>`__.
+The up-to-date Galaxy API documentation can be accessed by appending ``api/docs``
```

Comment:
Should /redoc be included as well?

Reply:
if we do that it should be hosted at `/api/redoc`

Reply:
If someone tells me how to change this (if easy :) ) then I can change it .. or just push to this branch. 

For me the most important point is that one (the one mentioned in the release notes) is documented.

Reply:
https://fastapi.tiangolo.com/tutorial/metadata/#docs-urls